sanitize/exempts.go

```diff
@@ -29,3 +30,6 @@ var entityLevelExemptedFields = map[string]map[string]struct{}{
 var configLevelExemptedFields = map[string]struct{}{
 	"ID": {},
 }
+
+// dynamically generated maps of exempted fields from schemas
+var entityLevelExemptedFieldsFromSchema = map[string]map[string]bool{}
```

Comment:
I see there are nil checks on this field - but is it explicitly getting marked as nil anywhere?
it initialises to a map, so

Reply:
Not marking it nil right now anywhere but it is a safety check and doesn't have any perf issues anyway. In future, if we mark it as nil, the check can help to avoid panics.